.github/workflows/blog-posts.yml

```diff
@@ -0,0 +1,22 @@
+name: Update Blog Posts
+
+on:
+  schedule:
+    - cron: '0 13 * * 1'
+  workflow_dispatch:
+
+jobs:
+  get_blog_posts:
+    name: Get Most Recent Posts from Bandwidth Blog
+    runs-on: ubuntu-latest
+    # outputs:
+      # release_needed: ${{ steps.release_needed.outputs.release_needed }}
+    steps:
+      - name: Get Blog Posts from Wordpress API
+        id: get_posts
+        run: |
+          BLOG_POSTS=$(curl -Ls -H "Authorization: token $WORDPRESS_TOKEN" https://www.bandwidth.com/wp-json/wp/v2/posts?per_page=20 | jq '[del(.[] | select(.categories[] | contains(1680))) | .[] | {imageId: .featured_media, categories: .categories, postLink: .link, postTitle: .title.rendered}] | .[:10]')
```

Comment:
does `.[:10]` return the first 10 items in the json array? why not just set `per_page=10` instead then?

Reply:
`per_page` defaults to 10, which would be fine, but we need to exclude the posts with the `Culture` tag, and if there's one in those 10, then we don't have enough blog posts to fill the carousel.